tools/cbmc/proofs/DNS/DNSgetHostByName/DNSgetHostByName_harness.c

```diff
@@ -0,0 +1,30 @@
+/* FreeRTOS includes. */
+#include "FreeRTOS.h"
+#include "queue.h"
+
+/* FreeRTOS+TCP includes. */
+#include "FreeRTOS_IP.h"
+#include "FreeRTOS_DNS.h"
+#include "FreeRTOS_IP_Private.h"
+#include "FreeRTOS_Sockets.h"
+
+/*This assumes fixed sized UDPPayloadBuffer and pcHostName. This also assumes that functions FreeRTOS_GetUDPPayloadBuffer, FreeRTOS_socket
+are implemented correctly. Also the prvParseDNSReply is considered to be correct as it is already proved.*/ 
+
+void * FreeRTOS_GetUDPPayloadBuffer(size_t xRequestedSizeBytes, TickType_t xBlockTimeTicks ) {
+	void *pvReturn = malloc(50);
+	return pvReturn;
+}
+
+Socket_t FreeRTOS_socket( BaseType_t xDomain, BaseType_t xType, BaseType_t xProtocol ) {
+	Socket_t xSocket = malloc(sizeof(Socket_t));
+	return xSocket;
+}
+
+uint32_t prvParseDNSReply( uint8_t *pucUDPPayloadBuffer, size_t xBufferLength, TickType_t xIdentifier) {}
+
+void harness() {
+	char *pcHostName = malloc(10);
```

Comment:
Same comment as [this](https://github.com/aws/amazon-freertos/pull/665#discussion_r288116679)